src/features/store.js

```diff
@@ -1,8 +1,12 @@
 import { configureStore } from '@reduxjs/toolkit';
-import rocketsSlice from '../redux/rockets/rocketsSlice';
+import missionsReducer from './missions/missionsSlice';
+import usersReducer from './users/usersSlice';
+import rocketsSlice from './rockets/rocketsSlice';
 
 const store = configureStore({
   reducer: {
+    missions: missionsReducer,
+    users: usersReducer,
     rockets: rocketsSlice,
```

Comment:
Please check the documentation on the use of Slice or Reducer in the store or if both can be used. While the component is displaying fine, I am not sure this won't throw an error going forward; but I may be mistaken. In the redux toolkit documentation, I could just see the 'componentReducer' syntax. Here is the link: https://redux-toolkit.js.org/usage/usage-guide/

Reply:
Thanks @diarisdiakite, I think, before merging, I should rename `rocketsSlice` to `rocketsReducer` in the `store.js` file so we can have the same naming syntax and respect best practices.